src/pages/auth.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { LeftPanel } from '../pagesComponents/Auth/LeftPanel';
+import { AuthLayout } from '../layouts';
+
+export default function Auth() {
+  const url = window.location.search.substring(1);
+  let title = '';
+  if (url === 'set-password') {
```

Comment:
Why are you using React Router if you're going to check your url in conditionals and get it from the window.location?